modules/ROOT/pages/getting-started.adoc

```diff
@@ -7,63 +7,99 @@ xref:glossary:index.adoc[glossary].
 == Key concepts
 
 === Namespace
-A namespace in a Kubernetes cluster can have sole or shared ownership. As a user, it is the most global
-resource that is generally available to you. Most of the Kubernetes Custom Resources (CRs) which you interact
-with are scoped to a single namespace including Components, Applications, Snapshots, and Secrets.
-
-All builds are performed by Tekton PipelineRuns within a namespace and usually driven by PipelineDefinitions
-committed to git repositories and triggered via https://pipelinesascode.com/[Pipelines as Code].
+In Kubernetes,
+link:https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/[namespaces]
+provide a foundational mechanism for isolating groups of resources within a
+single cluster. All of the {ProductName} resources and APIs that you interact
+with are scoped to namespaces, including your Components, Applications, Snapshots,
+Secrets and the Tekton PipelineRuns that perform builds, tests, and releases.
 
 ==== Tenant namespace
 These are namespaces where artifacts are produced from Tekton Pipelines. These can be accessed by more than one
-individual according to users' roles and the permissions defined by 
+individual according to their roles and the permissions defined by 
 link:https://konflux-ci.dev/architecture/ADR/0011-roles-and-permissions.html[those roles]. The tenant namespaces can be either
 for an individual or a team.
 //TODO: Document the process for getting access to/creating new namespaces
 
-You can grant, modify, or remove other users access to any namespace for which you have the `admin` role. Each
-namespace's admins have the responsibility to ensure that appropriate access is granted to users. If users are
-over-privileged, they may have access to any added secrets or to perform other unintended actions.
+In {ProductName}, you operate in a tenant namespace which is scoped to your
+team. One team can span multiple namespaces if needed, each with many
+applications with components. Namespaces should NOT be shared by many teams.
+
+In your tenant namespace, you will create Components and Applications, run the
+build PipelineRuns that are defined in your git repositories, view and iterate
+on the results of your IntegrationTestScenarios, and create Releases to release
+specific Snapshots according to specific ReleasePlans.
+
+NOTE: You can grant, modify, or remove other users' access to any namespace for
+which you have the `admin` role. Each namespace's admins have the
+responsibility to ensure that appropriate access is granted to users. If users
+are over-privileged, they might have access to any added secrets or to perform
+other unintended actions.
```

Comment:
Is this something that you want to call out here or should we call it out when talking about adding users? Part of the challenge that we have to contend with is information overflow when people are just getting started.